web/src/app/components/Modal/EditBannerModal.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+import { Box, Avatar, Divider, Flex, ActionIcon, Text, Image } from '@mantine/core';
+import styles from '../../styles/Modal.module.css';
```

Comment:
Please store the components module.css file in the relevant component folder. The Styles folder is for pages styles exclusively